doc/reference/tooling/tt_cli/cluster.rst

```diff
@@ -1,10 +1,242 @@
+
+.. _tt-cluster-centralized-authentication:
+
+Authentication
+~~~~~~~~~~~~~~
+
+There are three ways to pass the credentials for connecting to the centralized configuration storage:
+
+*   Environment variables ``TT_CLI_ETCD_USERNAME`` and ``TT_CLI_ETCD_PASSWORD``:
+
+    .. code-block:: console
+
+            $ export TT_CLI_ETCD_USERNAME=myuser
+            $ export TT_CLI_ETCD_PASSWORD=p4$$w0rD
+            $ tt cluster show "http://localhost:2379/myapp"
+
+*   ``tt cluster`` options ``-u``/``--username`` and ``-p``/``--password``:
+
+    .. code-block:: console
+
+        $ tt cluster show "http://localhost:2379/myapp" -u myuser -p p4$$w0rD
+
+*   Credentials specified in the storage URI: ``https://username:password@host:port/prefix``:
+
+    .. code-block:: console
+
+        $ tt cluster show "http://myuser:p4$$w0rD@localhost:2379/myapp"
+
+They are applied with the following precedence, from highest to lowest:
+
+*   URI credentials.
+*   ``tt cluster`` options.
+*   Environment variables.
+
+If connection encryption is enabled on the configuration storage, pass the required
+SSL parameters in the :ref:`URI arguments <tt-cluster-centralized-uri>`.
+
+.. _tt-cluster-centralized-uri:
+
+URI format
+~~~~~~~~~~
+
+A URI of the cluster configuration storage has the following format:
+
+.. code-block:: text
+
+    http(s)://[username:password@]host:port[/prefix][?arguments]
+
+*   ``username`` and ``password`` define credentials for connecting to the configuration storage.
+*   ``prefix`` is a base path identifying a specific application in the storage.
+*   ``arguments`` defines connection parameters. The following arguments are available:
+
+    *   ``name`` -- a name of an instance in the cluster configuration.
+    *   ``key`` -- a target configuration key in the specified ``prefix``.
+    *   ``timeout`` -- a request timeout in seconds. Default: ``3.0``.
+    *   ``ssl_key_file`` -- a path to a private SSL key file.
+    *   ``ssl_cert_file`` -- a path to an SSL certificate file.
+    *   ``ssl_ca_file`` -- a path to a trusted certificate authorities (CA) file.
+    *   ``ssl_ca_path`` -- a path to a trusted certificate authorities (CA) directory.
+    *   ``verify_host`` -- verify the certificate’s name against the host. Default ``true``.
+    *   ``verify_peer`` -- verify the peer’s SSL certificate. Default ``true``.
+
+.. _tt-cluster-instance:
+
+Managing configurations of specific instances
+---------------------------------------------
+
+In addition to whole cluster configurations, ``tt cluster`` can manage
+configurations of specific instances within applications. In this case, it operates
+with YAML fragments that describe a single :ref:`instance configuration section <configuration_overview>`.
+For example, the following YAML file can be a source when publishing an instance configuration:
+
+.. code-block:: yaml
+
+    # instance.yaml
+    iproto:
+      listen:
+      - uri: 127.0.0.1:3389
+      threads: 10
```

Comment:
Small suggestions regarding this code snippet, up to you whether to change or not:

- Rename `instance.yaml` -> `instance_source.yaml`. Adds some confusion with `instances.yaml`.
- Change `3389` to smth else because it is used for RDP access in Windows. And this adds unnecessary associations :) Maybe smth like `3311` or `33001` to show the difference with the default iproto port.
- Remove `threads` - looks like very specific use case.